lib/bridge/instance.js

```diff
@@ -0,0 +1,175 @@
+/*eslint no-invalid-this: 0*/ // eslint doesn't understand Promise.coroutine wrapping
+"use strict";
+var Promise = require("bluebird");
+var extend = require("extend");
+
+var store = require("../store");
+var ircToMatrix = require("./irc-to-matrix.js");
+var matrixToIrc = require("./matrix-to-irc.js");
+var membershiplists = require("./membershiplists.js");
+var ircLib = require("../irclib/irc.js");
+var names = require("../irclib/names.js");
+var IrcServer = require("../irclib/server.js").IrcServer;
+var matrixLib = require("../mxlib/matrix");
+var MatrixUser = require("../models/users").MatrixUser;
+var logging = require("../logging");
+var log = logging.get("IrcBridge");
+var Bridge = require("matrix-appservice-bridge").Bridge;
+
+const DELAY_TIME_MS = 10 * 1000;
+const DEAD_TIME_MS = 60 * 1000 * 5; // 5min
```

Comment:
5 * 60 * 1000 or 1000 * 60 * 5 would both be more clear than 60 * 1000 * 5, and probably wouldn't need that comment